libgenic/glass.c

```diff
@@ -91,7 +69,13 @@ setup_glass(double shift, int Ngrid, int seed)
 
     gsl_rng_free(rng);
 
+    evolve_glass(seed, ICP, NumPart);
+    return NumPart;
+}
 
+void evolve_glass(int seed, struct ic_part_data * ICP, const int NumPart)
```

Comment:
Perhaps mark this as static and remove it from the .h file, since it is not used outside this module.

Reply:
I have a feature I'm trying out that uses it in main.c If it doesn't turn out to be useful I'll remove this from the header

Reply:
If we are exposing more functions, then the functions in the module shall be properly prefixed ..

Reply:
glass_evolve ?

Reply:
Prefixed and some comments added